Makefile

```diff
@@ -21,7 +21,7 @@ all: binaries
 CFLAGS := -std=c99 -Wall -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA
 
 ifdef WITH_TCPDEVMEM_CUDA
-	CFLAGS += -DWITH_TCPDEVMEM_CUDA
+	CFLAGS += -DWITH_TCPDEVMEM_CUDA -I/kernel-includes/include
 endif
 ifdef WITH_TCPDEVMEM_UDMA
 	CFLAGS += -DWITH_TCPDEVMEM_UDMA -DNDEBUG=1 -static -I ~/cos-kernel/usr/include
```

Comment:
This patch fixes the build, so I would say it's pretty urgent, but if possible, lets also apply the changes to udmabuf.

So, since you need the header files to be copied in usr/ for cuda, lets also make that a requirement for udmabuf, to unify the build process.

The code change needed here would be to -Iusr/include/ like you do -I/kernel-includes/include above.

Reply:
done